### ISSUE TITLE
IQIYI_VID competition

### ISSUE THREAD

**Issue by nttstar** [deepinsight/insightface#439]
Recently www.iqiyi.com released a great video person dataset called [IQIYI_VID](http://challenge.ai.iqiyi.com/detail?raceId=5afc36639689443e8f815f9e) and also launched a person search competition on it. We finally got rank 1st(with team name WitcheR) by using Arcface models.  It is a very large and real dataset and worth trying to verify your face model accuracy precisely.  

~~Our solution will open source soon here after PRCV2018 conference.~~
You can download our code [here](https://pan.baidu.com/s/1d50eJ3efwpo7aadZU2wY6Q)

Development leaderboard is now available so anyone can submit your prediction to get your mAP. Leave messages here if you have any problem.

**Comment by xxllp** [comment]
想问下，现在的Megaface 有公司能达到99.9，这个用该项目能达到嘛

**Comment by gogo00007** [comment]
Can I ask only frame based info. is used in iqiyi task? Audio and temporal features have not been used?

**Comment by nttstar** [comment]
@gogo00007 right.

**Comment by AITTSMD** [comment]
@nttstar Hi, can you share the PPT?

**Comment by XupingZHENG** [comment]
@nttstar Hi, another request for your slides

**Comment by anguoyang** [comment]
is possible to share your pretrained model of iqiyi task?

**Comment by CStianya** [comment]
@nttstar Hi, may I ask what models a, c, d, e and h respectively represent? Where can I download it?

**Comment by symisc** [comment]
Cannot download the zip from Baidu. Could somebody share the zip archive (Drive, Dropbox or here) for me. Thanks guys
@nttstar @anguoyang @AITTSMD @XupingZHENG @xxllp @gogo00007 @doctorwgd @CStianya

**Comment by liuxcc** [comment]
@nttstar Hi, can you share the PPT or any other reference materials about this project? Some of the details are really hard to understand.

**Comment by prnvjb** [comment]
Can somebody share it in google drive or dropbox?
Thanks!!

**Comment by vlad3996** [comment]
@prnvjb https://github.com/deepinx/iqiyi-vid-challenge

**Comment by vlad3996** [comment]
> Can somebody share it in google drive or dropbox?
> Thanks!!

@prnvjb , if you ask about the code and pretrained models - check [this repo](https://github.com/deepinx/iqiyi-vid-challenge)

**Comment by prnvjb** [comment]
Thanks @vlad3996 !!

**Comment by chenshen03** [comment]
Does anyone use this code to test mAP on the IQIYI VID dataset? What is the result?

**Comment by chenshen03** [comment]
In addition, I found that this code uses the original softmax layer instead of the arcface softmax layer. Why?

**Comment by x-dragonet** [comment]
@nttstar,  thank you for sharing the code about 2018 iQIYI-VID challenge, would you mind share the pretrained model?
